SPECS/grpc/grpc.spec

```diff
@@ -0,0 +1,82 @@
+Summary:        Open source remote procedure call (RPC) framework
+Name:           grpc
+Version:        1.35.0
+Release:        1%{?dist}
+License:        ASL 2.0
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          Applications/System
+URL:            https://www.grpc.io
+Source0:        https://github.com/grpc/grpc/archive/v%{version}/%{name}-%{version}.tar.gz
+# A buildable grpc environment needs functioning submodules that do not work from the archive download
```

Comment:
I'm slightly confused by this message. It sounds like the source from the `Source0` tag is not the real source we're using to build this package. It seems to be the case, since the hash of the file downloaded from there doesn't match the one we've got in the signatures. If that's the case, please don't use a URL in the `Source0` tag, because that's just not the tarball we're actually using.

Reply:
Agree that having a URL in Source0 could be misleading here, though I'm questioning if this tarball (and other manually-created tarballs) are reproducible in the furst place. I just attempted to recreate the tarball using the steps and the resulting hash does not line up with the one in the signatures file.

One alternative could be to pull each of the underlying submodules in as tarballs from their upstream URL source and recreate the full source in the %prep step. It'd be a bit of a pain for projects like grpc which have 15 submodules but would allow us to rely on the signatures for both cached and remotely pulled sources.

Thoughts?

Reply:
I agree that we need to rework how we're constructing source tars that aren't a direct download from their respective project. I've created a task to track this and we can discuss it in when we sync next.

In the meanwhile I think that grpc is fine to go in as it is, I need the package available in the repo soon. We can revisit the source tar along with the other manually constructed ones all at once in the future.